### PR TITLE
[JBPM-9399] Add support to jBPM and runtime/cluster environments detection

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9399

process service start in clustered environment.

### CODE REVIEW

**Comment by elguardian** [comment]
depends on https://github.com/kiegroup/droolsjbpm-knowledge/pull/468

**Comment by afalhambra** [comment]
Jenkins do fdb

**Review by afalhambra — APPROVED** [review]
Only an unused import statement should be removed.
Nothing else other than that. It looks good to me.

File: jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Optional;
```
[the remark this answers is not in the document]
Comment (elguardian) [reply jbpm-flow/src/main/java/org/jbpm/process/instance/ProcessRuntimeImpl.java]: removed

**Comment by elguardian** [comment]
Jenkins retest this

**Pull request merged**